src/dataaggregator/db/db.go

```diff
@@ -0,0 +1,10 @@
+package db
+
+import (
+	"dataaggregator/policy"
+)
+
+type DB interface {
```

Comment:
If this is from metricscollector, then we should pull this out of both to src/db and share across them.